Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -1081,9 +1081,7 @@ public function testShardManager(): void
         $this->assertEquals($managerClass, $bazManagerDef->getClass());
     }
 
-    /**
-     * @requires PHP 8
-     */
+    /** @requires PHP 8 */
```

Comment:
This is reverting https://github.com/doctrine/DoctrineBundle/pull/1358. Is it intended ?

Reply:
No, it is not. Thanks for pointing this out!